packages/manager/fragments/navbar/src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const SECONDARY_UNIVERSES = ['partners'];
```

Comment:
I think this is no longer required as `Patners` universe no longer appears in navbar

Reply:
i removed the "partner" key in secondary_universes array but kept the array in case a secondary universe would need to be handled